### ISSUE TITLE
Merge with seisloc?

### ISSUE THREAD

**Issue by anowacki** [speedshi/MCM_MAT#1]
Maybe in time it would be good to move these codes into [seisloc](/speedshi/seisloc)?  That way we can ensure that the utility functions and migration code don't go out of sync, and users only need to check out one repo.

**Comment by speedshi** [comment]
Yes. That's a good idea. I will continue complete this codes: 1.add seismic format support for SAC file; 2. add the migration results analysis part. Once it is ready and complete for using, we can merge this one with the seisloc. 

One thing I'm in concern is this is just the Matlab version of integrating with MCM. If we merge that with seisloc, and one day we develop a python version, will that be a problem?

**Comment by anowacki** [comment]
> One thing I'm in concern is this is just the Matlab version of integrating with MCM. If we merge that with seisloc, and one day we develop a python version, will that be a problem?

I don't think so, since the Matlab stuff can always remain in the repo without causing any harm.  A future Python version which simply calls the Fortran executable would be easy to do, though a proper Python module which directly calls a compiled library would be significantly different and likely use Python's own multiprocessing rather than MPI within the compiled code.
